ASF-slaves.txt

```diff
@@ -27,12 +27,41 @@ Add ssh pub key from puppet deployment repo:
   $ sudo chmod 700 /home/jenkins/.ssh
   $ sudo chmod 600 /home/jenkins/.ssh/authorized_keys
 
+Install and Configure Puppet:
+    1. I added an 'asf999' user which has a pub key file containing keys from the Infra Team. (It has our standard uid/gid of 910)
+        (See also: https://github.com/apache/infrastructure-p6/blob/production/data/common/asf999.yaml )
+    2. The asf999 user has sudo no passwd (configured in visudo)
+    3. Recreated the Jenkins user account, see more below (in Option A)
```

Comment:
I'm not sure why Gavin had to recreate the jenkins account here, may want to ask him. At min, we should get rid of the reference to Option A (which I recall something we called out in the email thread and is not in these instructions)